src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,66 @@
 package com.theironyard.novauc;
+import java.util.ArrayList;
+import java.util.Scanner;
 
 public class Main {
+    public static ArrayList<InventoryItem> items = new ArrayList<>();
 
     public static void main(String[] args) {
-	// write your code here
+
+        Scanner scanner = new Scanner(System.in);
+        createitem();
+
+
+    }
+
+    public static InventoryItem createitem() {
+        while (true) {
+
+
+            Scanner scanner = new Scanner(System.in);
+
+            System.out.println("1. Create item");
+            System.out.println("2. List items");
+
+            String option = scanner.nextLine();
+
+
+            if (option.equals("1")) {
+                System.out.println("Enter your item name: ");
+                String item = scanner.nextLine();
+                System.out.println("Enter quantity: ");
+                int quantity = Integer.valueOf(scanner.nextLine());
+                System.out.println("Enter category: ");
+                String category = scanner.nextLine();
+
+                items.add(new InventoryItem(item, quantity, category));
+
```

Comment:
Jacob on line 36 you need to use the static method createItem to make a new InventoryItem.  Please add that method and resubmit.